addons/web/static/tests/views/basic_model_tests.js

```diff
@@ -1309,19 +1309,20 @@ odoo.define('web.basic_model_tests', function (require) {
                 res_id: 1,
             };
 
-            model.load(params).then(function (resultID) {
+            await model.load(params).then(function (resultID) {
```

Comment:
Long:
We added a `await Promise.resolve()` at the beginning of load, meaning that before doing anything, we waited for a tick. In these tests, we called `load`, and directly after we updated (in place) the params Object. As a consequence, the params of the first called were altered before being used.

Short:
What you did is correct.